### ISSUE TITLE
Compare offers of competitors

### ISSUE THREAD

**Issue by DajanSles** [DajanSles/GESPRO_Practica_2_Curso_2024_2025#42]
This part of the project compare offers of competitors.

**Comment by DavidSantaolalla** [comment]
> ¿Cómo obtendremos los datos de la competencia? ¿Necesitamos una fuente externa para la información de productos bancarios?

Correcto. Estamos en conversaciones con una empresa que proporciona datos financieros comparativos. Les mantendré informados sobre el progreso.

**Comment by DajanSles** [comment]
Genial, de esa manera podemos integrar los datos de la competencia en tiempo real si obtenemos acceso.
